__tests__/app.test.js

```diff
@@ -87,23 +88,52 @@ describe("GET /api/articles/:article_id", () => {
 });
 
 describe("GET /api/articles", () => {
-  test("200- should respond with an articles array containing article objects, each with the correct properties", () => {
+  test("200- should respond with an array of article objects, each with the correct properties", () => {
     return request(app)
       .get("/api/articles")
       .expect(200)
-      .then(({ body }) => {
-        console.log(body);
+      .then(({ body: { articles } }) => {
+        if (articles.length > 1) {
+          articles.forEach((article) => {
+            expect(article).toHaveProperty("article_id");
+            expect(article).toHaveProperty("title");
+            expect(article).toHaveProperty("topic");
+            expect(article).toHaveProperty("author");
+            expect(article).toHaveProperty("created_at");
+            expect(article).toHaveProperty("votes");
+            expect(article).toHaveProperty("article_img_url");
+            expect(article).toHaveProperty("comment_count");
+          });
+        }
+      });
+  });
+  test("should return articles sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles[0]).toEqual({
+          article_id: 3,
+          title: "Eight pug gifs that remind me of mitch",
+          topic: "mitch",
+          author: "icellusedkars",
+          created_at: "2020-11-03T09:12:00.000Z",
```

Comment:
we can use jest sorted for this :)

Reply:
Totally forgot about that! Thanks for the reminder